src/ray/core_worker/actor_handle.h

```diff
@@ -26,6 +26,7 @@
 namespace ray {
 namespace core {
 
+enum ActorStateSubscribeStatus { UNSUBSCRIBED, SUBSCRIBING, SUBSCRIBED };
```

Comment:
Why do we need both SUBSCRIBING and SUBSCRIBED?

Reply:
> Why do we need both SUBSCRIBING and SUBSCRIBED?

Before, I want to use SUBSCRIBING and SUBSCRIBED to distinguish that if the subscription is finished, because the actor name can only be cached after that.

Just as you suggested, if the actor name could be regenerated by actor handel, this enum does not need anymore.